tests/manage/pv_services/test_resource_deletion_during_pvc_pod_creation_and_io.py

```diff
@@ -94,10 +93,21 @@ class TestResourceDeletionDuringCreationOperations(ManageTest):
     pvc_size = 5
 
     @pytest.fixture()
-    def setup(self, interface, multi_pvc_factory, pod_factory):
+    def setup(self, request, interface, multi_pvc_factory, pod_factory):
         """
         Create PVCs and pods
         """
+        if config.ENV_DATA["platform"].lower() in constants.MANAGED_SERVICE_PLATFORMS:
+            # Get the index of current consumer cluster
+            self.consumer_cluster_index = config.cur_index
```

Comment:
The test cases will require consumer for workload. So the multicluster job can have a consumer as the primary cluster. So at the start of any such test case the cluster context will be consumer.
The number of test cases  which do not require consumer cluster is very less in ocs-ci. So I think those tests which need provider only can be selected to run on provider  rather than skipping other test cases on provider.

Reply:
Right. I have a suggestion. Maybe we can add two functions in the cluster file `ocs_ci/ocs/cluster.py`: 

- `is_ms_consumer_cluster` - which will returns `True` if it's a consumer cluster. `False`, otherwise.
- `is_ms_provider_cluster` - which will returns `True` if it's a provider cluster. `False`, otherwise.

Reply:
For the tests which require provider only, It will be nice if we can select such tests with some marker like "run_on_provider" or "provider_cluster_required" rather than adding skipif_ms_provider  in many other test cases.
Because most test cases in ocs-ci will need consumer.

Reply:
@fbalak Added skipif_ms_consumer as suggested.